### PR TITLE
update docs for local APB development

### PR DESCRIPTION
**Pull request by philbrookes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,3 +204,26 @@ Wait for all the pods to have come up, before proceeding (this can take a few mi
...
+### Developing APBs Locally
+
+Create a cluster pointing at your own docker organisation, this will also copy all the existing APBs in feedhenry to the dockerhub_org:
+```
+ansible-playbook playbook.yml -e "dockerhub_username=<dockerusername>" -e "dockerhub_password=<dockerpassword>" -e "dockerhub_org=<USE_THIS_VALUE>" --ask-become-pass
+```
+
+If the APB you wish to develop already exists in the feedhenry organisation, then syncing that APB can be skipped by adding:
+```
+-e "skip_apb=<apb-name>"
```
Comment (matzew) [inline README.md]: nice - is that new ?

File: README.md [hunk 2]
```diff
@@ -204,3 +204,26 @@ Wait for all the pods to have come up, before proceeding (this can take a few mi
...
+```
+
+Make your changes locally, and change the image name in the `apb.yml` file to match your organisation name. Then run:
+```
+make DOCKERORG=<your-org-name>
```
Comment (matzew) [inline README.md]: should be mentioned that this also pushes to "DOCKERHOST" ?

File: README.md [hunk 3]
```diff
@@ -204,3 +204,26 @@ Wait for all the pods to have come up, before proceeding (this can take a few mi
...
+```
+
+As the `apb.yml` has been updated, you will need to do a push, run:
+```
+apb push --broker=<asb-broker-route>
```
Comment (matzew) [inline README.md]: This is a push to the ASB, right ?   What's the difference between `bootstrap` ?

Comment (philbrookes) [reply README.md]: as I understand it, bootstrap re-scans the docker repo. push pushes just the apb.yml of the CWD

File: README.md (hunk 1 above)
Comment (philbrookes) [reply README.md]: yeah, I noticed a bunch of duplicates would show up if you were working on APB that was also cloned from feedhenry. It came in, in this PR: https://github.com/feedhenry/mcp-standalone/pull/142/files

File: README.md (hunk 2 above)
Comment (philbrookes) [reply README.md]: I've updated this now.

**Pull request merged**